### PR TITLE
Print matching token for operator types instead of plain integer

### PR DESCRIPTION
**Pull request by f4lco**
What is your opinion on the location of the stream operator on `OperatorType`? Currently it is an implementation detail of the utils. It would also possible to make the signature part of `Expr.h` to let other clients benefit from proper printing.
Should one be able to distinguish unary minus and binary minus?

### CODE REVIEW

File: src/util/sqlhelper.cpp [hunk 1]
```diff
@@ -237,4 +228,44 @@ namespace hsql {
     }
   }
 
+  std::ostream& operator<<(std::ostream& os, const OperatorType& op) {
+    static const std::map<const OperatorType, const std::string> operatorToToken = {
```
Comment (mrzzzrm) [inline src/util/sqlhelper.cpp]: maybe use a normal switch here? https://stackoverflow.com/questions/931890/what-is-more-efficient-a-switch-case-or-an-stdmap 

minor

Comment (f4lco) [reply src/util/sqlhelper.cpp]: Interesting read. I would like to argue that this particular method is not subject to performance concerns. It's a debug helper for printing a parse tree directly to `cout`. Even larger query plans will contain a modest number of operators.
In this case I would favor conciseness and readability over lookup-performance. I think creating this map only once accounts for the performance aspect adequately.

**Comment by f4lco** [comment]
OK I'll omit the commit with the generated files.

**Comment by f4lco** [comment]
I really appreciated if you answered the questions from the PR comment, even if it's just explicit consent to the current state.

File: src/util/sqlhelper.cpp (hunk 1 above)
Comment (mrks) [reply src/util/sqlhelper.cpp]: As `operator<<` should never be a performance-critical method, I don't think that it makes sense to spend time here.

**Comment by mrks** [comment]
Argh. No idea how this got lost. :(

**Comment by f4lco** [comment]
@mrks @mrzzzrm finally! 😆

**Pull request merged**